lib/fpm/package/deb.rb

```diff
@@ -710,6 +712,18 @@ def fix_dependency(dep)
     end
   end # def fix_dependency
 
+  def validate_dependencies()
+    self.dependencies.each do |dep|
+      if (dep !~ /^[A-Z]/ && dep !~ /^[a-z]/)
+        logger.warn("Dependency '#{dep}' is not a valid deb dependency " \
```

Comment:
fpm will [convert to lowercase[(https://github.com/jordansissel/fpm/blob/master/lib/fpm/package/deb.rb#L234-L239) the package name if it contains uppercase letters.

I worry that removing the dependency isn't the correct action. Do you have any thoughts on lowercasing the dependency instead?